static/src/js/components/ManageCollectionAssociation/__tests__/__mocks__/manageCollectionAssociationResults.js

```diff
@@ -137,13 +141,297 @@ export const toolRecordSearch = {
     }
   }
 }
+export const toolRecordSearchwithPages = {
+  request: {
```

Comment:
I like having these long mocks in its own file, I think future PRs, I may start pulling them out.